controllers/lvmcluster_controller.go

```diff
@@ -46,6 +46,7 @@ type LVMClusterReconciler struct {
 }
 
 //+kubebuilder:rbac:groups=lvm.topolvm.io,resources=lvmclusters,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=apps,resources=daemonsets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
This should be part of the "add vgmanager" commit.

Reply:
done